### PR TITLE
Add minutes from today's meeting (2019-09-03)

### PR DESCRIPTION
**Pull request by kierdavis**

### CODE REVIEW

**Review by TomWheal — APPROVED** [review]

File: 2019/meeting-2019-09-03.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        * In progress, turns out team maintainers don't work expected. Kier will carry on chasing this up with the org admins and trustees
+    * Kier will create Github repo for KTC minutes and tasks (#6)
+        * Done
+    * KTC will resolve whether they want to approve the kit team addition to the ops manual via email (#9)
+        * Rob to kick-start this discussion
+    * Tom will email the people who have signed up so far (#7)
+        * Not done yet
+* Motion 0: motion-passing arrangement
+    * Could use github for formalising approval process
+        * We'll consider this next meeting
```
Comment (rgilton) [inline 2019/meeting-2019-09-03.md:49]: Perhaps add an action point for this?

File: 2019/meeting-2019-09-03.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    * All in favour, motion passed
+* Trustee key dates
+    * "Present plans for next kit development cycle"
+        * We would like development itself to be continuous
+        * However this is about plans - what are we planning to work on over the next few months?
+    * All looks good
+    * KD to tell trustees it all looks good
+* Should non-team-members have permission to participate in discussions on team repos (e.g. non-KTC on KTC minutes repo)?
+    * Some people in favour of closed discussions for improved team cohesion
```
Comment (rgilton) [inline 2019/meeting-2019-09-03.md]: Clarification on "closed discussions": Publicly readable, not publicly writable.

File: 2019/meeting-2019-09-03.md (hunk 1 above)
Comment (kierdavis) [reply 2019/meeting-2019-09-03.md:49]: Added to next week's agenda.

File: 2019/meeting-2019-09-03.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+* (ongoing) Tom will email the people who have signed up so far (#7)
+* Rob will initiate discussion about whether we want to approve the kit team addition to the ops manual (#9)
+* Kier will tell trustees that we've reviewed the key dates doc and it looks fine
+* Tom and Rob will get container key from Rich (#10)
+* Tom and Kier will examine the inventory and estimate how many kits we can provide (#11)
+* Rob to write an email to the trustees about committee roles and brainstorming session (#12)
+
+## Appendices
+
+### Motion 1: Github permissions
```
Comment (rgilton) [inline 2019/meeting-2019-09-03.md]: Perhaps add a note to say this wasn't passed?

Comment (trickeydan) [reply 2019/meeting-2019-09-03.md]: It wasn't rejected either. We determined that we needed to discuss it further at a later date.

Comment (kierdavis) [reply 2019/meeting-2019-09-03.md]: Moved it to a "Motions Not Passed" section.

**Review by rgilton — APPROVED** [review]

File: 2019/meeting-2019-09-03.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+     * A motion proposed via email, using the process described above.
+     * A motion voted on at a meeting, provided that it is accepted by more than 75% of the _KTC_ (not just the quorum for standard decisions).
+
+## Motions Not Passed
+
+### Motion 1: Github permissions
+
+Proposed by Rob.
+
+Background: It seems that non-trustee members have admin rights over our repositories.  This will get in the way of any plans we have regarding the autonomy of sub-teams, and indeed our own autonomy.  The current arrangement has presumably come about due to Github only recently growing "child teams", and the ability to give someone permission to create child teams.
```
Comment (RealOrangeOne) [inline 2019/meeting-2019-09-03.md:143]: As said non-trustee member with admin rights, I'd be interested to know how I will "get in the way of any plans"? I really don't want that!

(Unsure if this is the best place to discuss?)

**Pull request merged**